src/views/CreateEventView.vue

```diff
@@ -88,17 +106,43 @@ export default {
         description: String,
         whatToBring: String,
       */
+      imageUploaded: 0,
       event: {},
       error: '',
     };
   },
   methods: {
+    convertImage(file) {
```

Comment:
Can you write some comments explaining how these functions work in case someone else needs to modify this in the future? Cause I really don't get how this works lol.

Reply:
Where is the image actually uploaded to s3? Or are you storing it as a blob in the database?

Reply:
I'll write in some comments explaining the process

Reply:
also not sure how complex/specific this is, but if it would be challenging to implement somewhere else. There is an npm package for this, aws-sdk.

Reply:
I added some comments that try to explain whats going on. The image gets encoded, sent to the backend, and the web server handles upload to S3.